packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -110,21 +125,28 @@ export class RemoteChannelContext implements IChannelContext {
         this.services.deltaConnection.setConnectionState(connected);
     }
 
+    public rebaseOp(message: ISequencedDocumentMessage, localOpMetadata: unknown) {
+        if (this.isLoaded) {
```

Comment:
Left code in another comment about same, but I think rebaseOp should be async and should force load particular channel. Delaying it till connection does not buy us anything and makes code harder to follow